Public/Src/Engine/Processes/FileAccessManifest.cs

```diff
@@ -851,6 +851,32 @@ public ArraySegment<byte> GetPayloadBytes(FileAccessSetup setup, MemoryStream st
             }
         }
 
+        /// <summary>
+        /// Explicitly releases most of its memory.
+        /// </summary>
+        internal void Release()

```

Comment:
No noticeable difference on selfhost, but FAMs are not big there.  I'll check bigger builds later.

Reply:
But it does seem noticeable in bigger builds